

class PyErr:
    def __init__(self):
        return